packages/hooks/src/useCFDRealAccounts.ts

```diff
@@ -0,0 +1,22 @@
+import { useStore } from '@deriv/stores';
+import useCFDAccounts from './useCFDAccounts';
+
+const useCFDRealAccounts = () => {
+    const { traders_hub } = useStore();
+    const { is_eu_user } = traders_hub;
+    const accounts = useCFDAccounts();
+
+    const cfd_real_accounts = accounts.filter(account => {
+        const is_demo = account.account_type === 'demo';
```

Comment:
move to enum or constant

Reply:
@ali-hosseini-deriv Everything is type-safe already! 😕